h2o-r/h2o-DESCRIPTION.template

```diff
@@ -26,6 +26,7 @@ Imports: graphics,
 Suggests: ggplot2,
           mlbench,
           Matrix,
+          slam,
```

Comment:
Yes, that is exactly what I did, a warning for users without slam and data.table.